src/StateManager.ts

```diff
@@ -76,6 +77,7 @@ export class StateManager {
   static setFolderState(folder: string, key: KEYS.FEATURE_CONFIGURATIONS, value: Record<string, FeatureConfiguration[]> | undefined): Thenable<void>
   static setFolderState(folder: string, key: KEYS.ENVIRONMENTS, value: Record<string, Environment> | undefined): Thenable<void>
   static setFolderState(folder: string, key: KEYS.ORGANIZATION, value: Organization | undefined): Thenable<void>
+  static setFolderState(folder: string, key: KEYS.ORGANIZATIONS, value: Record<string, Organization> | undefined): Thenable<void>
```

Comment:
the organizations wouldn't need to be scoped by folder, since the orgs available to the current user will be the same for all folders